specification/network/resource-manager/readme.md

```diff
@@ -451,6 +481,17 @@ batch:
   - tag: package-2015-06split
 ```
 
+### Tag: package-2018-01 and python
+
+These settings apply only when `--tag=package-2018-01 --python` is specified on the command line.

```

Comment:
@lmazuel This look cool? Note `namespace`/`output-folder` have 2017-...

Reply:
not cool indeed, thanks for being careful!
In this tag, all 2017_11_01 needs to be updated to 2018_01_01

Reply:
@marstr Please review this PR for Go multiapi

Reply:
@marstr @lmazuel before spending too much time here, note [my comment](https://github.com/Azure/azure-rest-api-specs/pull/2271#issuecomment-360929670), this PR should ideally just be merged into the existing network 2018 PR (if it still exists)